terraform-provider-constellation/internal/provider/attestation_data_source.go

```diff
@@ -108,10 +83,6 @@ func (d *AttestationDataSource) Schema(_ context.Context, _ datasource.SchemaReq
 				MarkdownDescription: "For Azure only, the URL of the Microsoft Azure Attestation service",
 				Optional:            true,
 			},
-			"id": schema.StringAttribute{
-				Computed:            true,
-				MarkdownDescription: "The ID of the data source",
-			},
 			"measurements": schema.MapNestedAttribute{
```

Comment:
I think this is superfluous now, as it is contained in the `attestation` output, right?